src/Api/Droplet.php

```diff
@@ -93,18 +93,19 @@ public function getById(int $id)
      * @param string|int   $image
      * @param bool         $backups
      * @param bool         $ipv6
-     * @param bool         $privateNetworking
+     * @param bool         $privateNetworking Deprecated, use vpcUuid
```

Comment:
Better to rename this parameter and have it accept `bool|string`.

Reply:
No problem, I'll make this change. To confirm it should be `bool|string $privateNetworking` or `bool|string $vpcUuid`?